packaging/vendor/multitech/conduit-ap3/files/concentratord.toml

```diff
@@ -48,8 +48,9 @@ model=""
 # Gateway vendor / model flags.
 model_flags=[]
 
-# Gateway ID.
+# Override Gateway ID.
```

Comment:
This should be equal to the `configfile.rs` template:

```toml
# Gateway ID.
#
# Only set this if you would like to override the Gateway ID provided by the SX1302/3.
```

Reply:
Done.